docs/how-tos/manage-modules.md

```diff
@@ -62,7 +62,15 @@ If you intend to make frequent code changes to your module, want to support a va
    tar -czf module.tar.gz <PATH-TO-EXECUTABLE>
    ```
 
-   Where `<PATH-TO-EXECUTABLE>` is your [packaged executable](/how-tos/create-module/#compile-or-package-your-module).
+   where `<PATH-TO-EXECUTABLE>` is the [packaged executable](/how-tos/create-module/#compile-or-package-your-module).
+
+   For a Python module built using the `venv` approach, the command might look like this:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   tar -czf module.tar.gz run.sh requirements.txt src
+   ```
+
+   Where `run.sh` is your [entrypoint file](/how-tos/create-module/#compile-or-package-your-module), `requirements.txt` is your [pip dependency list file](/how-tos/create-module/#compile-or-package-your-module), and `src` is the directory that contains the source code of your module.
```

Comment:
This can be confusing (I might be confused) but I believe what we initially refer to as the entrypoint file is not run.sh, it is the main program file -- see [write a module](https://docs.viam.com/how-tos/create-module/#write-a-module). Maybe just refer to it as the executable that runs your program?

Reply:
maybe we just don't even need this explanation...